examples/jaeger-mixin/README.md

```diff
@@ -0,0 +1,27 @@
+# Prometheus monitoring mixin for Jaeger
```

Comment:
At first, I thought `examples` would be an appropriate location for this, but now, I'm not so sure anymore, as this is an actual library that people import into a `jsonnetfile.json`, like:

```json
{
    "dependencies": [
        {
            "name": "jaeger-mixin",
            "source": {
                "git": {
                    "remote": "https://github.com/jaegertracing/jaeger",
                    "subdir": "examples/jaeger-mixin"
                }
            },
            "version": "master"
        }
    ]
}
```

How about a top-level directory called "monitoring"? Another option is to have a separate repository for those files, but I think there's benefit in having this close to where most of the metrics are defined (ie, in this repo), so that these files can be changed whenever there's a change related to metrics.

@objectiser, @yurishkuro thoughts?

Reply:
If it is intended to be more than just an example, then yes should be moved - and "monitoring" or "dashboards" might be appropriate.

Reply:
Dashboard is one of the possible outputs of this library. Alerts is the other one. So, "monitoring" would be more appropriate, IMO